calliope/test/test_core_util.py

```diff
@@ -166,32 +166,19 @@ def test_memoize_instancemethod(self):
 
 
 class TestLogging:
```

Comment:
You can easily test this with logging.getLogger('calliope').getEffectiveLevel(), whereby critical = 50 down to debug = 10. So i'd reinstate this test ;)

Reply:
Fixed